lib/fabrik/VirtualHostInstance.js

```diff
@@ -22,6 +23,12 @@ class VirtualHostInstance extends BaseInstance {
       .then(deploymentName => this.deploymentName = deploymentName);
   }
 
+  get platformContext() {
+    return Promise.resolve({
```

Comment:
wrong. Should get the parent instance id and from there can get the deployment name and from the deployment can figure out the context. THis is incorrect.

Reply:
changed

Reply:
The virtual host related change we haven't done as it has cloud controller specific operations and we have decided not to do changes. That's why was returning default value. Now I remember.

Reply:
Actually returning no ops is ideal.